src/components/form/custom-fields/course-syllabus.tsx

```diff
@@ -0,0 +1,79 @@
+import { ChangeEvent, useState } from 'react';
+import { TypographyScale } from '@mdb/flora';
+import styles from './styles';
+
+const CourseSyllabusField = ({
+  setValue,
+}: {
+  setValue: (val: string | File) => void;
+}) => {
+  const [field, setField] = useState<'fileUpload' | 'webUrl' | ''>('');
+  const [inputFocus, setInputFocus] = useState(false);
+
+  function onUploadDocClick() {
+    setField('fileUpload');
+  }
+
+  function onEnterUrlClick() {
+    setField('webUrl');
+  }
+
+  return (
+    <div>
```

Comment:
Do we need this `div` here, or could we use a react fragment `<>`?

Reply:
yeah need this block element for layout of children